MobiFlight/MobiFlightCache.cs

```diff
@@ -86,6 +90,27 @@ public void Start()
                     } 
                 });
         }
+        
+        // Calls SetPowerSaveMode(false) on all connected modules. If the
+        // force parameter is set to true then all modules will be sent the
+        // command even if the keep awake interval hasn't passed yet.
+        public void KeepConnectedModulesAwake(bool force = false)
+        {
+            // lastKeepAwake is initialized to the earliest possible DateTime so the first
+            // time this method is called this test will fail and the modules will be forced
+            // to set their power save mode to off.
+            if (!force && lastKeepAwake.AddMinutes(KeepAwakeIntervalInMinutes) >= DateTime.UtcNow)
+            {
+                return;
+            }
+
+            foreach (var module in Modules)
+            {
+                module.Value.SetPowerSaveMode(false);

```

Comment:
See my comment in the firmware too, based on your naming here `KeepAwake` describes better what this is about.

Reply:
See my comment on the firmware repo 😂